src/ast/mod.rs

```diff
@@ -346,6 +346,12 @@ pub enum Expr {
     },
     /// POSITION(<expr> in <expr>)
     Position { expr: Box<Expr>, r#in: Box<Expr> },
+    /// LOCATE(<expr>, <expr>, <expr>)
```

Comment:
Wouldn't the parser parse `LOCATE("foo", "foobar")` already today?

The `POSITION` has some special syntax (`IN`) but `LOCATE` looks like a normal function 

So I am wondering why this PR is needed

Reply:
I confirmed that it is parsed as a function because it does not need such a special keyword and is separated only by commas. As you said, the PR seems unnecessary. I will close this PR.